src/explore-education-statistics-frontend/src/middleware/pages/redirectPages.ts

```diff
@@ -0,0 +1,62 @@
+import redirectService, {
+  Redirects,
+  RedirectType,
+} from '@frontend/services/redirectService';
+import type { NextRequest } from 'next/server';
+import { NextResponse } from 'next/server';
+
+interface StoredRedirects {
+  redirects: Redirects;
+  fetchedAt: number;
+}
+
+let storedRedirects: StoredRedirects | undefined;
+
+const redirectPaths = {
+  methodologyRedirects: '/methodology',
```

Comment:
Given the owning object is called `redirectPaths`, these attributes can probably just be called `methodologies` and `publications`.

Reply:
This is my naming inherited from the backend. I'll change the name in the backend to `methodologies`.

Reply:
Changed to `methodologies` and `publications` following the back end change.

Reply:
Nice one, thanks!